tests/golden/column-name-_fields.csv

```diff
@@ -0,0 +1,4 @@
+_fields,foo,bar,baz

```

Comment:
Is it reasonable to combine these into a single test?  Or do they have to be the first column?

Reply:
They do not have to be! A single test would be less laborious to scale. The reason why i thought separate tests made sense was bc if it regresses, their individual error messages are not very clear. But it is a lot of files and I really am fine either way.

Reply:
Exactly. The different files cover the different ways the column names can conflict in the implementation. While they could all be put into one file, it would make debugging a regression harder. In fact, I started my local debugging that way, but it wasn't immediately clear when I fixed one way because the others still failed.